### ISSUE TITLE
Fix get_var_itemsize implementation

### ISSUE THREAD

**Issue by mdpiper** [nhm-usgs/bmi-prms6-surface#12]
The *get_var_itemsize* function should return the size, in bytes, of one element of a BMI variable.

For example, the code for `nlake` is correct because it's a scalar, but for variables of BMI grid type "vector", like `hru_ppt`, the code should be
```
size = sizeof(this%model%model_simulation%model_precip%hru_ppt(1))
``` 
to get just one element of the variable.

**Comment by rmcd-mscb** [comment]
Thanks Mark - See Pull request

**Comment by mdpiper** [comment]
@rmcd-mscb fixed this with #13.
